tests/tests/fv3_ccpp_gsd_RRTMGP

```diff
@@ -0,0 +1,125 @@
+###############################################################################
+#
+#  FV3 CCPP GSD RRTMGP (GF CU + Thompson MP + MYNN PBL + RUC LSM + RRTMGP) full (0-48h) test
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 CCPP GSD RRTMGP results with previous trunk version"
```

Comment:
I see two gsd_RRTMGP tests are added, but they are not added to rt.conf?

Reply:
I opened an issue for this so that we can move on: https://github.com/ufs-community/ufs-weather-model/issues/335